nautobot/extras/tests/test_jobs.py

```diff
@@ -670,3 +674,55 @@ def test_missing_receive_job_hook_method(self):
         name = "TestJobHookReceiverFail"
         job_result = create_job_result_and_run_job(module, name, data=self.data, commit=False)
         self.assertEqual(job_result.status, JobResultStatusChoices.STATUS_ERRORED)
+
+    def test_inherited_field_order(self):
+        module = "test_job_hook_receiver"
+        name = "TestJobHookReceiverChange"
+        job_class, job_model = get_job_class_and_model(module, name)
+        form = job_class().as_form()
+        self.assertHTMLEqual(
+            form.as_table(),
+            """<tr><th><label for="id_object_change">Object change:</label></th><td>
+            <select name="object_change" class="nautobot-select2-api form-control form-control" display-field="display" required placeholder="None" data-url="/api/extras/object-changes/" id="id_object_change">
+            <option value="" selected>---------</option></select></td></tr>
+            <tr><th><label for="id_a_testvar_b_first">A testvar b first:</label></th><td>
+            <input type="text" name="a_testvar_b_first" class="form-control form-control" required placeholder="None" id="id_a_testvar_b_first"></td></tr>
+            <tr><th><label for="id_a_testvar_a_second">A testvar a second:</label></th><td>
+            <input type="text" name="a_testvar_a_second" class="form-control form-control" required placeholder="None" id="id_a_testvar_a_second"></td></tr>
```

Comment:
This feels weird, since it feels like a JobHookReceiver shouldn't even permit the definition of extra vars.

Reply:
This was just to test the field order for variables inherited from the base class. I'll move it to a regular job test